File: frontend/projects/api/src/lib/events/event-summary.ts
import { LocalTimeInput } from './local-time-input';

export interface EventSummary {
  timezone: string | null;
  start: LocalTimeInput | null;
  end: LocalTimeInput | null;
  startsAtUtc: string | null;
  endsAtUtc: string | null;
  id: string;
  title: string | null;
  published: boolean;
  useLiturgy: boolean;
  version: string;
}

File: backend/src/FaithTechTorontoAiBuildEvent.Application/Events/EventSummary.cs
namespace FaithTechTorontoAiBuildEvent.Application.Events;

public sealed record EventSummary(Guid Id, string? Title, bool Published, bool UseLiturgy, string Version,
    string? Timezone = null, LocalTimeInput? Start = null, LocalTimeInput? End = null,
    DateTimeOffset? StartsAtUtc = null, DateTimeOffset? EndsAtUtc = null);
